pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc

```diff
@@ -0,0 +1,24 @@
+[id="module-metadata-in-pantheon_{context}"]
+= Module metadata in {ProductShortName}
+
+Each module in {ProductShortName} has associated metadata for search engine optimization (SEO). SEO improves the findability of modules.  
+
+There are two types of module-metadata in {ProductShortName}:
+
+* Intrinsic:
++
+The metadata that is inferred from the module automatically.
++
+Example of intrinsic metadata include the following:
+
+** Module name
+** Module description
+
+* Extrinsic:
++
+The metadata that is applied manually before publishing.
++
+Examples of extrinsic metadata include the following:
+
+** Product name - the product and the product-version that the module should be published for.
```

Comment:
The hyphen is unnecessary in "product version".

Reply:
removed hyphen